src/v/kafka/server/snc_quota_manager.cc

```diff
@@ -19,6 +19,8 @@ using namespace std::chrono_literals;
 
 namespace kafka {
 
+using quota_t = snc_quota_manager::quota_t;
```

Comment:
Closer to 15 places actually, and only a few of them are return types of member functions.

I have decided that it makes sense because `quota_t` makes sense for the enire unit and this definition causes no ambiguity in its scope.

Reply:
except for the return values, it seems like most quota_t has a type definition that is available in scope because it already defines on the snc_quota_manager::quota_t.

Reply:
> except for the return values, it seems like most quota_t has a type definition that is available in scope because it already defines on the snc_quota_manager::quota_t.

there are many namespace functions dealing with it too

Reply:
ok